pkg/agent/multicast/mcast_controller.go

```diff
@@ -513,8 +515,10 @@ func (c *Controller) syncGroup(groupKey string) error {
 			if err := c.igmpSnooper.sendIGMPLeaveReport([]net.IP{group}); err != nil {
 				klog.ErrorS(err, "Failed to send IGMP leave message to other Nodes", "group", groupKey)
 			}
+			klog.InfoS("Sent the IGMP leave message to other Nodes", "group", groupKey)
 		}
 		c.delInstalledLocalGroup(groupKey)
+		klog.InfoS("Removed local multicast group", "group", groupKey)
```

Comment:
IMO, all the new `klog.InfoS` introduced in this PR should not have default verbosity

Reply:
updated with klog.V(2)